.travis.yml

```diff
@@ -17,19 +17,22 @@ addons:
         # used in MUC + s2s tests combination
         - muc.localhost
 before_install:
+        - pip install --user awscli
+        - if [ $PRESET != "pkg" ]; then tools/travis-download-build-artifacts-from-s3.sh; fi
         # Do not execute this step, if environment variable SKIP_RELEASE is "1"
         # Configure is a part of release creation
         - test 1 = "$SKIP_RELEASE" || tools/configure $REL_CONFIG
 install:
         - test 1 = "$SKIP_COMPILE" || travis_retry ./rebar3 get-deps
         - test 1 = "$SKIP_COMPILE" || ./rebar3 compile
+        - test 1 = "$SKIP_COMPILE" || make certs
         - test 1 = "$SKIP_RELEASE" || make devrel
         - test 1 = "$SKIP_BUILD_TESTS" || tools/travis-build-tests.sh
         - test 1 = "$SKIP_COV" || travis_retry pip install --user codecov
 before_script:
         - tools/travis-setup-db.sh
         - if [ $PRESET = 'ldap_mnesia' ]; then sudo tools/travis-setup-ldap.sh; fi
-script: KEEP_COVER_RUNNING=1 tools/travis-test.sh -p $PRESET
+script: KEEP_COVER_RUNNING=1 tools/travis-test.sh -p $PRESET -s $RUN_SMALL_TESTS
```

Comment:
I'm just not sure - how this `RUN_SMALL_TESTS` variable is comparable to `SMALL_TESTS` from `travis-test.sh` here https://github.com/esl/MongooseIM/blob/master/tools/travis-test.sh#L6

Reply:
It's passed as value of `-s` parameter to `travis-test.sh` script. Then in https://github.com/esl/MongooseIM/blob/master/tools/travis-test.sh#L10-L33 it get's parsed and overrides the value of `SMALL_TESTS` form line 6.

Reply:
Oh, now I get it. 
It would be easier to grep if we had the same var name in `travis-test.sh` :D